onnxruntime/core/optimizer/layout_transformation/layout_transformation.cc

```diff
@@ -162,7 +162,8 @@ Status TransformLayoutForEP(Graph& graph, bool& modified, const IExecutionProvid
       // Except for resize and convolution ops, all the other layout sensitive ops only require layout transformation
       // for 0th input and output. For resize, add the other relevant inputs which need conversion. For Conv - layout
       // transformer only converts layout for 0th input, weights should be handled by every EP.
-      if (node->OpType() == "Resize") {
+      // For resize in WebNN EP, we don't want to convert all the inputs except the 0th input.
+      if (node->OpType() == "Resize" && node->GetExecutionProviderType() != kWebNNExecutionProvider) {
```

Comment:
I'm not sure I'm the right person to sign off for this file, but I see you added satyajandhyala, who presumably touched this file before. It's interesting that I haven't heard of any issues in the DML EP related to this, and so I'm assuming this layout transformation is unilaterally disabled for the DML EP.